elements/rh-subnav/rh-subnav.ts

```diff
@@ -0,0 +1,202 @@
+import { LitElement, html } from 'lit';
+import { classMap } from 'lit/directives/class-map.js';
+import { customElement } from 'lit/decorators/custom-element.js';
+import { query } from 'lit/decorators/query.js';
+import { queryAssignedElements } from 'lit/decorators/query-assigned-elements.js';
+import { property } from 'lit/decorators/property.js';
+
+import { isElementInView } from '@patternfly/pfe-core/functions/isElementInView.js';
+import { RovingTabindexController } from '@patternfly/pfe-core/controllers/roving-tabindex-controller.js';
+
+import { colorContextConsumer, type ColorTheme } from '../../lib/context/color/consumer.js';
+import { colorContextProvider, type ColorPalette } from '../../lib/context/color/provider.js';
+
+import '@patternfly/elements/pf-icon/pf-icon.js';
+
+import styles from './rh-subnav.css';
+
+/**
+ * Subnav provides a tabs-like navigation experience
+ * @slot - Place navigation links here
```

Comment:
Should we also document CSS vars?

Reply:
So there are no custom css properties for specifically for subnav only csspart, I added those.